kaldb/src/main/java/com/slack/kaldb/chunkManager/IndexingChunkManager.java

```diff
@@ -343,7 +348,7 @@ protected void startUp() throws Exception {
     stopIngestion = false;
 
     directorySizeExecutorService.scheduleAtFixedRate(
-        () -> approximateDirectoryBytes = FileUtils.sizeOf(dataDirectory),
+        () -> approximateDirectoryBytes = FileUtils.sizeOf(activeChunkDirectory),
```

Comment:
Please add a unit test to test this change. Also, add a unit test to ensure that the current bug is fixed. 

The gauge still needs to be reset on a roll over otherwise I don't think the bug is fixed.

Reply:
By gauge you mean `long approximateDirectoryBytes` ? It doesn't hurt to reset it, but strictly speaking I don't think it's needed? i.e if you see where we calculate it the reset will happen automatically on the next document add.

I will address it anyways